geode-core/src/main/java/org/apache/geode/internal/cache/UpdateAttributesProcessor.java

```diff
@@ -462,7 +462,7 @@ public static void send(InternalDistributedMember recipient, int processorId,
       if (exception != null) {
         m.setException(exception);
         if (logger.isDebugEnabled()) {
-          logger.debug("Replying with exception: {}" + m, exception);
+          logger.debug("Replying with exception:{}", m, exception);
```

Comment:
I wonder if the intent was for just the exception message here and not the stack.

Reply:
I think it is safe as it is inside debug flag